lib/browser/objects-registry.js

```diff
@@ -47,10 +47,12 @@ class ObjectsRegistry {
   // Dereference an object according to its ID.
   remove(webContentsId, id) {
     // Dereference from the storage.
-    this.dereference(id);
+    const noReferences = this.dereference(id);
 
-    // Also remove the reference in owner.
-    this.owners[webContentsId].delete(id);
+    if (noReferences) {
```

Comment:
When `remove` is called, it should be guaranteed that the object with `id` is already referenced. I think we should fix the root cause instead of working around the bug.